pom.xml

```diff
@@ -3728,12 +3729,11 @@
               <report>dependency-info</report>
               <report>dependency-management</report>
               <report>index</report>
-              <report>issue-management</report>
-              <report>licenses</report>
-              <report>mailing-lists</report>
+              <report>issue-tracking</report>
+              <report>mailing-list</report>
```

Comment:
Did report names change? No licenses?

Reply:
Yes, some reports were renamed similarly to the generated html files.
I removed `licenses` because we don't use license.html any more but have a link to https://www.apache.org/licenses/ which is an ASF site requirement. This improvement was done by @symat in HBASE-20405.